frontend/webadmin/modules/uicommonweb/src/main/java/org/ovirt/engine/ui/uicommonweb/models/storage/UploadImageModel.java

```diff
@@ -245,6 +245,8 @@ public void initialize() {
             if (imageInfoModel.getContentType() == DiskContentType.ISO) {
                 getDiskModel().getAlias().setEntity(imageInfoModel.getFileName());
                 getDiskModel().getDescription().setEntity(imageInfoModel.getFileName());
+                diskModel.getIsIncrementalBackup().setEntity(false);
+                diskModel.getIsIncrementalBackup().setIsChangeable(false);
```

Comment:
please check if it changes to `changeable` again when picking another image that is not ISO (so you open the dialog, select an ISO so it gets disabled and then picking a different image that is not of type ISO)